### DISCUSSION TITLE
Can not install react-hook-form

### DISCUSSION THREAD

**Discussion by ab-siddiq** [react-hook-form/react-hook-form#8290] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/24230435/166163992-2f7a83df-a649-4c7f-afaa-2bcbead58f46.png)
$ npm install react-hook-form
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: @react-aria/ssr@3.1.2
npm WARN Conflicting peer dependency: react@17.0.2
npm WARN node_modules/react
npm WARN   peer react@"^16.8.0 || ^17.0.0-rc.1" from @react-aria/ssr@3.1.2
npm WARN   node_modules/@restart/ui/node_modules/@react-aria/ssr
npm WARN     @react-aria/ssr@"^3.0.1" from @restart/ui@1.2.0
npm WARN     node_modules/@restart/ui
npm ERR! code ERR_SSL_DECRYPTION_FAILED_OR_BAD_RECORD_MAC
npm ERR! errno ERR_SSL_DECRYPTION_FAILED_OR_BAD_RECORD_MAC
npm ERR! Invalid response body while trying to fetch https://registry.npmjs.org/react-hook-form: 3696:error:1408F119:SSL routines:ssl3_get_record:decryption failed or bad record mac:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:677:  
npm ERR!

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\User\AppData\Local\npm-cache\_logs\2022-05-01T20_24_47_904Z-debug-0.log

**Comment by Moshyfawn** [comment, marked as answer]
Are you under some kind of firewall? Maybe you're using Docker?

It's not an issue with RHF

```bash
$ npm install react-hook-form                

added 1 package, and audited 525 packages in 774ms

223 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

Looks very similar to nodejs/help#3815

**Comment by ab-siddiq** [comment]
npm set strict-ssl false

npm config set registry http://registry.npmjs.org/

npm config set ca=""

run above command which i got after searching error code and apply. But didn't work instantly. But after restarting my device it worked fine. Couldn't understand which solution actually worked and why this error happened. Can you please elaborately say about this problem so that i can get a clear understanding about this error.

**Comment by kassio-matheus** [comment]
### I had the same problem, and i found the solution, try this:

1. Uninstall the `node_modules`
2. After uninstall, paste `npm i` in your terminal
3. Enjoy :)

**_If the same problem continues after following my steps, try reinstalling Node.js_**

**Comment by oluseguncodess** [comment]
npm error code E404
npm error 404 Not Found - GET https://registry.npmjs.org/react-hook-form - Not found
npm error 404
npm error 404  'react-hook-form@*' is not in this registry.
npm error 404
npm error 404 Note that you can also install from a
npm error 404 tarball, folder, http url, or git url.

Why can't i install?
